views/pages/github.hbs

```diff
@@ -160,6 +160,20 @@
                 Compatibility implementation of the ECMAScript Internationalization API (ECMA-402) for JavaScript. <b>Developed by <a href="https://github.com/andyearnshaw">Andy Earnshaw</a></b>
             </p>
         </div>
+
+        <div class="repo">
+            <h3 class="repo-name">
+                <a href="https://github.com/yahoo/intl-locales-supported">Intl Locales Supported</a>
+            </h3>
+            <div class="repo-badges">
+              {{#with "intl-locales-supported"}}
+                {{> npm-badge}}
+              {{/with}}
+            </div>
+            <p class="repo-description">
+                Utility to help you polyfill the Node.js runtime when the Intl APIs are missing, or if the built-in Intl is missing locale data that you need.
```

Comment:
This is a copy/paste of the npm package's `"description"` field, like all the other packages listed on this page. Ideally we'd switch this to actually _using_ the `package.json` data.